scripts/test/e2e/run

```diff
@@ -15,7 +15,11 @@ function fetch_images {
 
 function setup {
     local project=$1
-    COMPOSE_PROJECT_NAME=$1 COMPOSE_FILE=$2 docker-compose up --build -d >&2
+    local file=./e2e/compose-env.yaml
```

Comment:
Previously `$file` would (logically) have been `$2` but here you seem to have expanded/inlined it without changing the caller AFAICT.